frontend/src/services/fetchData.ts

```diff
@@ -0,0 +1,70 @@
+import { AppActlApiService } from '@/services/ApiServiceFactory';
+import { setApplicationsUserAdministers } from '@/store/ApplicationState';
+import type { FamApplicationUserRoleAssignmentGet } from 'fam-app-acsctl-api';
+
+// --- Fetching data (from backend)
```

Comment:
Hi Ian, this file looks good to me, but I wonder if we could have another name? This is like a central place to make service calls including get and delete. I know in Redux these are called actions. Maybe something like appActlApiActions?  I'm really bad at naming ....

Reply:
I don't use Redux before and don't know what actions does. I think it is for state management.
I don't prefer terms stick to a particular technology stacks unless the terms really speaks to itself, like "store/state".
If other people use different stack then it isn't clear what "actions" do to them and may confuse others.
The fetchData is created just for convenience on taking care of the calls (really, it is just to fetching the data from using the api(s) ) many components could face and does not limit to just one API (unless we like to). I basically just like to fetch data from database (and some minor/general processing I see fit after data is fetched), does not involve state if it does not need to.